### PR TITLE
add SessionId method call

### PR DESCRIPTION
**Pull request by petroOv-PDFfiller**

### CODE REVIEW

File: src/test/java/org/selenide/selenoid/SelenoidClientTest.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ void encodesFileNameInUrl() throws MalformedURLException {
         new URL("http://localhost:4444/download/sid-01/some-file.txt")
     );
     assertThat(client.urlOfDownloadedFile("some file (2).txt")).isEqualTo(
-        new URL("http://localhost:4444/download/sid-01/some%20file%20(2).txt")
+        new URL("http://localhost:4444/download/sid-01/some%20file%20%282%29")
```
Comment (petroOv-PDFfiller) [inline src/test/java/org/selenide/selenoid/SelenoidClientTest.java:32]: I'm not sure about this, but i dont think that is caused by SessionId changes

**Review by asolntsev — COMMENTED** [review]
LGTM

**Comment by petroOv-PDFfiller** [comment]
Sorry, i forget to add description. This pull should fix problem with EventFiringDriver which provides ClassCastException. I described this problem in the pull request in base selenide repo.

**Comment by asolntsev** [comment]
@petroOv-PDFfiller Thank you! Release `selenide-selenoid:1.1.3`.

**Pull request merged**